### DISCUSSION TITLE
Quick Export: Checkbox field values always coming through as 'no'

### DISCUSSION THREAD

**Discussion by duncanmcclean** [solspace/craft-freeform#713] [Questions & Answers] [answered]
**Describe the bug or issue you're experiencing**

When you try to do a 'Quick Export' of submissions for a form which includes checkbox fields, the value of checkbox fields seems to be returning 'no', even though the user checked it when submitting the form & I can see it has been checked when viewing submissions individually. 

**Steps to reproduce**

1. Create a form with a checkbox field (we have 3 checkbox fields, all marked as required)
2. Submit it a few times to get a few submissions
3. Click 'Quick Export' and select all the checkbox fields. I'm exporting to "Excel" format.
4. See that the values of the checkbox fields are 'no'

**Expected behavior**

That the checkbox fields say 'yes' in the report because they are, welll... checked on the form submission.

**Craft & Plugin Info (please complete the following information):**

- Craft Version: Craft Pro 4.4.14
- Freeform Version: 4.1.0
- Freeform Edition: Pro
- Fresh Install or Upgrade: Upgrade from 3

**Additional context**

**Comment by kjmartens** [comment, marked as answer]
Hi @duncanmcclean,

When I test this on my end, it appears to work fine. Can you confirm if perhaps the checkbox fields contain a **Value** of `no` for them inside the form builder? If so, this is expected behavior, as Freeform will store the specified value for it. This is to maximize compatibility with other third-party systems, e.g. `Yes`, `yes`, `y`, `true`, etc. 🙂

**Comment by duncanmcclean** [comment]
Ah, that's the issue then. My value for that field was set to `no`.
